cartridges/int_cloudinary/cartridge/scripts/jobs/steps/cldUpdateProductImagesForFeeds.js

```diff
@@ -115,6 +116,13 @@ function writeProductFileContent(cloudinaryUrlStreamWriter, productSearchHitsItr
                         for (var k = 0; k < product.variants.length; k++) {
                             variantsID = product.variants[k].ID;
                             colorAttrValueID = cloudinaryHelper.fetchVariationAttrValueId(variantsID, cloudinaryConstants.COLOR_ATTR);
+
+                            if (!clrAttrArray.includes(colorAttrValueID)) {
+                                clrAttrArray.push(colorAttrValueID);
+                            } else {
+                                break;
```

Comment:
I think that continue is the right behavior, because if we use break we might miss some variants

Reply:
we only having variant once, because for other size variants it has the same image, otherwise it duplicates the products. I have verified and it works fine

Reply:
a customer might have a different configuration with different images per size.
also, there could be color variants, which we already have different images

Reply:
then we might change the break statement with continue? are you aligned with this?

Reply:
yes, please change it